coalib/output/ConsoleInteraction.py

```diff
@@ -907,7 +917,8 @@ def show_bear(bear,
 def print_bears(bears,
                 show_description,
                 show_params,
-                console_printer):
+                console_printer,
+                args=None):
```

Comment:
Here, `args`  param is super set of `show_description` and 'show_params`. I've not removed these 2 params so that public API is not broken. It should be removed at a later point of time.